lib/context.js

```diff
@@ -0,0 +1,3 @@
+const { createContext } = require('react');
+
+module.exports = createContext();
```

Comment:
Perhaps not for this PR, but we could expose a hook to grab the context value. Although I'm not really sure if that API is better than what we have now.

Reply:
If we add a hook interface, would that require another version bump? If so, maybe better to do it now?

Reply:
That would be additive so should be fine as a minor. So long as we don't take away the current API.

Reply:
At some point it might be worth the effort to convert the internals to use hooks, too, but that's probably overkill for this PR

Reply:
💯